transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/specification/ws/acceptor/LimitsIT.java

```diff
@@ -0,0 +1,162 @@
+/**
```

Comment:
Given my comment on the k3po PR asking you to make the test cases on use http://localhost:8080/echo, you will need to split this class into multiple ones, one per required different gateway configuration. I would recommend splitting into classes with names like "LimitsMaxSize126IT",  "LimitsMaxSizeDefaultIT", and "LimitsSessionTimeoutIT"